ReactiveCocoa/UIKit/iOS/UIRefreshControl.swift

```diff
@@ -13,8 +13,19 @@ extension Reactive where Base: UIRefreshControl {
 		return makeBindingTarget { $0.attributedTitle = $1 }
 	}
 
-	/// A trigger that sends `next` when the user performs a refresh.
-	public var refresh: Signal<Void, NoError> {
-		return trigger(for: .valueChanged)
+	/// The action to be triggered when the refresh control is refreshed. It
+	/// also controls the enabled state of the refresh control.
+	public var refreshed: CocoaAction<Base>? {
```

Comment:
That's a valid point. But for the moment, we also have stuff that are not named accordingly, e.g. `UIButton.pressed`, inherited from Rex. It seems we do need to figure out which way to go, hmm.

/cc @ReactiveCocoa/reactivecocoa